tips-for-getting-your-next-developer-job.rst

```diff
@@ -0,0 +1,183 @@
+.. post:: February 13, 2019
+   :tags: advertising, sustainability, hiring
+   :author: David
+   :location: SAN
+
+
+Tips For Getting Your Next Developer Job
+========================================
+
+Over the last month, the Read the Docs team conducted 30-40 customer development interviews
+with hiring managers and recruiters
+at companies ranging from 5-person companies to the biggest names in tech.
+We wanted to learn more about hiring processes at various companies
+with the ultimate goal of building `a product to help companies find developers`_.
+
+Last time, we covered some :doc:`tips for hiring managers <lessons-from-hiring-manager-interviews>`
+based on what companies told us they were doing.
+This time, we put together **tips for candidates looking for their next job**
+based on insights we heard from hiring managers.
+
+Rather than covering how to crack developer interviews which is its own challenge
+or how to gain the necessary technical skills,
+these tips will focus on getting your foot in the door. Specifically:
+
+* Making your application stand out
+* Improving your application response rate
+* Getting to the interview process
+
+.. _a product to help companies find developers: https://readthedocs.org/sustainability/advertising/recruiting/?utm_medium=referral&utm_source=readthedocs-blog&utm_campaign=next-developer-job
+
+
+.. figure:: img/2019-facebook-job-opening.png
```

Comment:
This feels a bit out of context, needs some transition or something from the bullet list -> image

Reply:
Ya, perhaps it's better as a segue to the "Spend a little more time on your application" section.

Reply:
Agree. I think the image is good because it gives you the feeling that the hiring person has to review hundred of applications and you need to differentiate yourself from those. It applies better on the other section, though.